packages/admin/src/models/proposal.ts

```diff
@@ -41,3 +59,18 @@ export interface ProposalFunctionInputType {
   internalType?: string;
   components?: ProposalFunctionInputType[];
 }
+
+export interface Token {
+  name: string;
+  symbol: string;
+  address: Address;
+  network: Network;
+  decimals: number;
+  type: 'ERC20';
+}
+export interface NativeCurrency {
+  name: string;
+  symbol: string;
+  decimals: number;
+  type: 'native';
+}
```

Comment:
Where did these come from?

Reply:
Also the backend, it should be in the response, we just were not typing it on the client